lib/java/com/google/android/material/slider/BaseSlider.java

```diff
@@ -2748,6 +2748,9 @@ protected void onPopulateNodeForVirtualView(
         contentDescription.append(startOrEndDescription(virtualViewId));
         contentDescription.append(slider.formatValue(value));
       }
+
+      contentDescription.append(" Value is" + slider.formatValue(value));
```

Comment:
Thanks for your feedback, do you mean we could add a check to support API 26 and below versions?

Reply:
Let's not worry about the API level thing. What I meant here is that we announce "range start, 2" here for range slider already. If we add this line here, it will now announce "range start, 2, value, 2" for range slider. I think at least we should put this line in the `else` block of the `if` above. So that it will announce "range start, 2" for range slider, and "value, 2" for regular slider.